frontend/src/components/Map.jsx

```diff
@@ -0,0 +1,39 @@
+import { MapContainer, TileLayer, Marker, Popup } from "react-leaflet";
+import useGeolocation from "react-hook-geolocation";
+import "leaflet/dist/leaflet.css";
+
+const shopPosition = [49.258329, 4.031696];
+
+function Map() {
+  const userGeolocation = useGeolocation();
+  const userPosition = [userGeolocation.latitude, userGeolocation.longitude];
+
+  return (
+    <div className="flex flex-col text-center">
+      <h1>Résultats :</h1>
+      {!userGeolocation.error && userGeolocation.latitude && (
```

Comment:
this should condition the user marker only, not the whole map ;)

also try to declare `userPosition` like this :

    const position = (geolocation) => {
      if (geolocation.error || !geolocation.latitude || !geolocation.longitude) {
        return null;
      }

      return [geolocation.latitude, geolocation.longitude];
    }

    function Map() [
      const userGeolocation = useGeolocation();
      const userPosition = position(userGeolocation);

      // ...

this will allow your test to become :

    {userPosition && <Marker position={userPosition}> ...}

Reply:
```
center={userPosition ?? someDefaultValue}
```